### DISCUSSION TITLE
23.10.17 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#31] [Daily Scrum]
.

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 소셜 로그인 이후 사용자가 요청했던 `originalUri`로 redirect 하는 방법 고민
    - 코드 리뷰 대기
- 내일은 어떤 일을 하나요?
    - 코드 리뷰를 반영하여 코드 수정

**Comment by kdmstj** [comment]
- 오늘은 어떤 일을 했나요?
    - 플레이리스트 CRUD 테스트 코드 작성 
- 내일은 어떤 일을 하나요?
    - 민주님 코드 리뷰
    - 플레이리스트 CRUD 테스트 코드 작성
    - PR

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - 트랜잭션 관련 테스트 케이스 추가
- 내일은 어떤 일을 하나요?
  - 트랜잭션 관련 테스트 디버깅
  - 스레드 확인하여 비동기 작업하도록 변경
